app/views/ReportView/components/CopyNumber/index.tsx

```diff
@@ -86,7 +86,7 @@ const CopyNumber = ({
             for (const {
               gene: {
                 expressionVariants: {
-                  tpm, rpkm, primarySiteFoldChange, diseasekIQR,
+                  tpm, rpkm, primarySiteFoldChange,
```

Comment:
in the other tables did diseaseZScore get added here or no?

Reply:
This is old code that forced display of diseasKIQR because it was by default not shown, so removing this ends up not showing kIQR by default